apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/question/request/CreateQuestionRequest.java

```diff
@@ -35,4 +38,100 @@ public record MessagingInfo(
                         String hl7DataType) {
         }
 
+        public enum CodedQuestionType {
+                SINGLE_SELECT(1007, "Single-Select (Drop down)"),
+                MULTI_SELECT(1013, "Multi-Select (List Box)"),
+                READONLY_SINGLE_SELECT_SAVE(1024, "Readonly single-select save"),
+                READONLY_MULTI_SELECT_SAVE(1025, "Readonly multi-select save"),
+                READONLY_SINGLE_SELECT_NO_SAVE(1027, "Readonly single-select no save"),
+                READONLY_MULTI_SELECT_NO_SAVE(1028, "Readonly multi-select no save"),
+                CODE_LOOKUP(1031, "Code Lookup");
+
+                private final long key;
+                private final String value;
+
+                CodedQuestionType(long key, String value) {
+                        this.key = key;
+                        this.value = value;
+                }
+
+                public static List<DisplayOption> getDisplayOptions() {
+                        return Arrays.stream(values()).map(CodedQuestionType::toDisplayOption).toList();
+                }
+
+                private DisplayOption toDisplayOption() {
+                        return new DisplayOption(key,value);
+                }
+        }
+
+        public enum DateQuestionType {
+                USER_ENTERED(1008, "User entered text, number, or date"),
+                READONLY_USER_ENTERED(1026, "Readonly User entered text, number, or date"),
+                READONLY_USER_NO_SAVE(1029, "Readonly User text, number, or date no save");
+                private final long key;
+                private final String value;
+
+                DateQuestionType(long key, String value) {
+                        this.key = key;
+                        this.value = value;
+                }
+
+                public static List<DisplayOption> getDisplayOptions() {
+                        return Arrays.stream(values()).map(DateQuestionType::toDisplayOption).toList();
+                }
+
+                private DisplayOption toDisplayOption() {
+                        return new DisplayOption(key,value);
+                }
+        }
+
+        public enum NumericQuestionType {
```

Comment:
Suggestion: What do you think about renaming these to `NumericDisplayControl` as the value maps to the `displayControl` field.

Also, what do you think about moving the various enum's into the question requests? For example, moving this `NumericQuestionType` into the `CreateNumericQuestionRequest.java` class